### DISCUSSION TITLE
Is there a way to make the keyboard more suitable for a round screen?

### DISCUSSION THREAD

**Discussion by Jarne47** [florisboard/florisboard#2114] [Q&A]
Hi, I am looking for an option to implement a keyboard on a round screen. I tried to change the layout through the settings to be more round-screen compatible, but I didn't find a suitable solution.

The Gboard for WearOS is a good example, can we customise the keyboard to look something like this:
![image](https://user-images.githubusercontent.com/96826144/202200610-271baf4c-cb9c-413f-8872-2aff579f02db.png)

I appreciate any help you can provide.

**Comment by patrickgold** [comment]
Thanks for the idea!

There is currently no direct way to build a layout in a round shape, but sometime in the future a WearOS support (and thus a specific UI layout optimized for Smartwatches) is planned. (There's a feature request for this IIRC)
